src/components/RecipeSubmit.vue

```diff
@@ -0,0 +1,297 @@
+                                <md-layout>
+                                    <md-input-container :md-inline="true">
+                                        <label>MM</label>
+                                        <md-input type="number" v-model="prepTimeMin" required></md-input>
+                                    </md-input-container>
+                                </md-layout>
+                            </md-layout>
+                        </md-layout>
+                        <md-layout :mdRow="true" :md-gutter="16">
+                            <span class="label">Cook Time:
+                                <md-avatar>
+                                    <md-icon class="help-icon">help</md-icon>
+                                    <md-tooltip class="primary">The amount of time food for which food is actually cooked.</md-tooltip>
+                                </md-avatar>
+                            </span>
+                            <md-layout :md-gutter="16">
+                                <md-layout>
+                                    <md-input-container :md-inline="true">
+                                        <label>HH</label>
+                                        <md-input type="number" v-model="cookTimeHr" required></md-input>
+                                    </md-input-container>
+                                </md-layout>
+                                <md-layout>
+                                    <md-input-container :md-inline="true">
+                                        <label>MM</label>
+                                        <md-input type="number" v-model="cookTimeMin" required></md-input>
+                                    </md-input-container>
+                                </md-layout>
+                            </md-layout>
+                        </md-layout>
+                        <md-input-container>
+                            <label>Servings</label>
+                            <md-input type="number" v-model="servings" required></md-input>
+                        </md-input-container>
+
+                        <v-select v-model="cookware" label="Required cookware" chips tags :items="cookwareOptions"
+                                  :hide-selected="true" :no-data-text="'No suggestions'"
+                              hint="Enter a piece of equipment (e.g. &quot;Rolling pan&quot;) and press Tab to record it." persistent-hint>
+                        </v-select>
+
+                        <!-- TODO: Tags -->
+
+                    </md-layout>
+                </md-layout>
+                <md-button type="submit" class="md-raised md-primary">Submit Recipe</md-button>
+            </form>
+        </div>
+    </div>
+    </v-app>
+</template>
+
+<script>
+  import Firebase from 'firebase';
+  import db from '../database';
+  import router from 'vue-router';
+  import uuidV4 from 'uuid/v4';
+  import VSelect from "vuetify/es5/components/VSelect/VSelect";
```

Comment:
Replace all double quotes literals with single quotes literals, unless you're specifically writing a string that contains a single quote.